sosw/components/helpers.py

```diff
@@ -394,6 +395,29 @@ def validate_date_from_something(d):
     return validate_datetime_from_something(d).date()
 
 
+def validate_date_timestamp_from_something(timestamp):
```

Comment:
Don't use the variable name `timestamp`. It will confuse with the datetime module.
And the argument is not a timestamp in most cases anyway.